src/pro/javacard/gp/Hash.java

```diff
@@ -0,0 +1,51 @@
+package pro.javacard.gp;
+
+import apdu4j.HexUtils;
+
+public class Hash {
```

Comment:
At first sight it seems like this class could be avoided and HexStringMatcher used instead.